ddtrace/internal/telemetry/writer.py

```diff
@@ -310,11 +307,11 @@ def add_error(self, code, msg, filename, line_number):
         self._error = (code, msg)
 
     def _app_started(self, register_app_shutdown=True):
-        # type: (bool) -> None
+        # type: (bool) -> Dict[str, Any]
         """Sent when TelemetryWriter is enabled or forks"""
         if self._forked or self.started:
             # app-started events should only be sent by the main process
-            return
+            return {}
```

Comment:
why not `None`? we can save on creating a dict just to test that it is not truthy

Reply:
I can update the type to Optional[Dict[str, Any]]. This method does not queue telemetry events any more. It only generates a payload. This refactor allowed me to avoid acquiring the periodic service lock multiple times in `TelemetryWriter.periodic()`.

Reply:
yeah, I think `Optional` is more in-line with what you did for other functions here as well?